### ISSUE TITLE
FileSelect 

### ISSUE THREAD

**Issue by ipeshev** [telerik/kendo-angular#3762]
... by user ticket

**Describe the bug**

We got following WCAG errors reported by AXE
 
-     Form elements must have labels: the form label is pointing to the div that looks like a button instead of the hidden field inside of it.
-     Interactive controls must not be nested: this appears to be the result of role="button" being added to the <div> that houses the hidden <input>.  It should be noted that if that code is removed then all of the associated aria attributes will also need to be removed.

**To Reproduce**

Open following [FileSelect demo](https://www.telerik.com/kendo-angular-ui/components/uploads/fileselect/) and inspect it with AxeDevTools or other AXE tool.

**Expected behavior**
It should not have WCAG violations
